content/en/_index.md

```diff
@@ -42,7 +42,7 @@ Learn more
 <section class="home--top-section why-grpc">
 <h2>Why gRPC?</h2>
 
-gRPC is a modern open source high performance RPC framework that can
+gRPC is a modern open source high performance Remote Procedure Call (RPC) framework that can
```

Comment:
Any concerns about line length? The rest of this paragraph is wrapped sooner.  Same below.

Reply:
I wanted to keep the diff in this PR as simple as possible.

I'll cleanup the line wrapping in followup "noop" PR -- i.e., a PR that has no effect on the generated site (as confirmed by a diff over the generated site files). Hopefully that's ok with you.

Reply:
Of course, I can line-wrap the paragraphs in _this_ PR. Let me know what you prefer @dfawley.

Reply:
I don't feel too strongly, but it seems better to do everything at once.

Also I would prefer the caps but, again, don't feel strongly.